### ISSUE TITLE
trim_suffix and trim_prefix methods not working

### ISSUE THREAD

**Issue by natenho** [benthosdev/benthos#1832]
Hi,

## Expected

[trim_prefix](https://www.benthos.dev/docs/guides/bloblang/methods/#trim_prefix)
[trim_suffix](https://www.benthos.dev/docs/guides/bloblang/methods/#trim_suffix)

```
root.name = this.name.trim_prefix("foobar_")
root.description = this.description.trim_prefix("foobar_")

# In:  {"description":"unchanged","name":"foobar_blobton"}
# Out: {"description":"unchanged","name":"blobton"}
```

## Actual

```
root.name = this.name.trim_prefix("foobar_")
root.description = this.description.trim_prefix("foobar_")

# In:  {"description":"unchanged","name":"foobar_blobton"}
# Out: failed to parse mapping: line 1 char 23: unrecognised method 'trim_prefix'
  |
1 | root.name = this.name.trim_prefix("foobar_")
  |                       ^---
```

![image](https://user-images.githubusercontent.com/4236481/229568202-735d4797-5f6c-4551-84df-4960e46ec24c.png)
![image](https://user-images.githubusercontent.com/4236481/229567466-dedcb19a-5f19-4b64-abe9-173657e0af25.png)

## Workaround

Use trim() instead if applicable

**Comment by mihaitodor** [comment]
Hey @natenho, which version of Benthos are you using? These were added in [v4.12.0](https://github.com/benthosdev/benthos/blob/main/CHANGELOG.md#4120---2023-02-20)

**Comment by natenho** [comment]
@mihaitodor, nevermind, I was using 4.11 - we should consider adding the "introduced in version" to the docs.

**Comment by mihaitodor** [comment]
> we should consider adding the "introduced in version" to the docs.
Good point, that's on me 😢 Fixed in #1833
